src/PostgREST/Auth.hs

```diff
@@ -196,6 +207,25 @@ getTimeSpec res maxLifetime utc = do
     Just (JSON.Number seconds) -> Just $ TimeSpec (sciToInt seconds - utcToSecs utc) 0
     _                          -> Just $ TimeSpec (fromIntegral maxLifetime :: Int64) 0
 
+calcApproxCacheSizeInBytes :: C.Cache ByteString AuthResult -> IO Int
```

Comment:
It would be good to add a comment on how this works. Maybe the one on https://github.com/PostgREST/postgrest/pull/3802#issuecomment-2506584190.